deployments.json

```diff
@@ -557,8 +557,8 @@
       "address": "0xCB829B1Aa77B8B57D320AF05a780757c8c2B88C1"
     },
     "sAFU": {
-      "txHash": "0x5856ef56a67575ed46d74f86e7c5c6a4eb344a516ddfdc83fac445b324ac96ac",
-      "address": "0xc7B4BB7c8e3620A6c4F9E96524ccB8a81D52A1b1"
+      "txHash": "0x9f05b29a297527d56a4d3baed643928d4267404acedd1aec6190ec72376fcc69",
+      "address": "0x97638479ee4a25e69F8782a18831325C78485565"
```

Comment:
diff: https://www.diffchecker.com/EWSNKK0L/ (but you should check for yourself with [smartdiffer](https://pypi.org/project/smartdiffer/))